src/lib/gundb/UserStorageClass.js

```diff
@@ -2004,8 +2032,9 @@ export class UserStorage {
           data: {
             endpoint: {
               address: sender,
-              fullName,
-              avatar,
+              fullName:
+                !fullNameFromProfile || fullNameFromProfile === 'Unknown' ? fullNameFromEvent : fullNameFromProfile,
```

Comment:
what if fullnamefromevent is empty?

Reply:
good question, should fullName have the value of "Unknown" in that case?

Reply:
yes

Reply:
done